protocol/synthetix/test/integration/modules/core/VaultModule.test.ts

```diff
@@ -19,6 +19,8 @@ describe('VaultModule', function () {
     collateralAddress,
   } = bootstrapWithStakedPool();
 
+  const MAX_UINT = ethers.BigNumber.from(2).pow(256).sub(1);
```

Comment:
even easier way: `ethers.constants.Uint256Max` (or something with a name close to that)